src/values.rs

```diff
@@ -660,3 +671,46 @@ fn check_division_by_zero(num: i32) -> Result<()> {
         _ => Ok(()),
     }
 }
+
+#[derive(Debug, Clone, PartialEq)]
+pub enum TransformerValue {
```

Comment:
Hmmm... it is common to name it `Transformer`, just the same as the variant name `Value::Transformer`

Reply:
there is a also a `Transformer` in parser/macro.rs, is it ok to keep this Value variant sturct and AST struct having same name `Transfomer`?

Reply:
Yes, they are in different mods